code/map.dm

```diff
@@ -31,6 +31,7 @@ var/global/list/mapNames = list(
 	"1 pamgoC" = 		list("id" = "PAMGOC", 		"settings" = "pamgoc", 			"playerPickable" = FALSE),
 	"Kondaru" = 		list("id" = "KONDARU", 		"settings" = "kondaru", 		"playerPickable" = TRUE,		"MaxPlayersAllowed" = 80),
 	"Ozymandias" = 	list("id" = "OZYMANDIAS", "settings" = "ozymandias", 	"playerPickable" = FALSE, 	"MinPlayersAllowed" = 40),
+	"Nadir" = 		list("id" = "NADIR", 		"settings" = "nadir", 		"playerPickable" = FALSE),
```

Comment:
Probably "MaxPlayersAllowed" = 80?

Reply:
Sounds like a sensible enough cap from how the tests played out.

Reply:
While I'm in there, should I enable player pickable so the map is available for rotation after merge, or do I stick to the current approach of the rotation activation being a separate action?